### PR TITLE
test: Add Sysman VF Management CTS

### PR DESCRIPTION
**Pull request by vishnu-khanth**
Related-To: VLCLJ-2312

### CODE REVIEW

File: conformance_tests/sysman/test_sysman_vf_management/src/test_sysman_vf_management.cpp [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+
+      for (const auto &vf_handle : vf_handles) {
+        EXPECT_NE(vf_handle, nullptr);
+      }
+    } else {
+      LOG_INFO << "No VF handles found for this device!";
+    }
+  }
+
+  if (!is_vf_enabled) {
```
Comment (pratikbariintel) [inline conformance_tests/sysman/test_sysman_vf_management/src/test_sysman_vf_management.cpp:78]: Is this condition required ?

Comment (vishnu-khanth) [reply conformance_tests/sysman/test_sysman_vf_management/src/test_sysman_vf_management.cpp:78]: yes. This is added to track whether vf is enabled on at least one device. After iterating through all devices, if vf is not enabled on any of the devices, then failing the test

File: conformance_tests/sysman/test_sysman_vf_management/src/test_sysman_vf_management.cpp [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+    uint32_t count = lzt::get_vf_handles_count(device);
+    if (count > 0) {
+      is_vf_enabled = true;
+      LOG_INFO << "VF is enabled on this device!!";
+      uint32_t test_count = count + 1;
+      auto vf_handles = lzt::get_vf_handles(device, test_count);
+      EXPECT_EQ(count, test_count);
+
+      for (int i = 0; i < count; i++) {
+        EXPECT_NE(vf_handles[i], nullptr);
```
[the remark this answers is not in the document]
Comment (vishnu-khanth) [reply conformance_tests/sysman/test_sysman_vf_management/src/test_sysman_vf_management.cpp:128]: Ok. Will update this.

[the remark this answers is not in the document]
Comment (vishnu-khanth) [reply conformance_tests/sysman/test_sysman_vf_management/src/test_sysman_vf_management.cpp]: Updated. Please check @pratikbariintel

**Review by pratikbariintel — DISMISSED** [review]

**Review by AshwinKumarKulkarni — APPROVED** [review]

**Pull request merged**